src/features/device-state/routes/state.ts

```diff
@@ -25,6 +26,163 @@ export const varListInsert = (varList: EnvVarList, obj: Dictionary<string>) => {
 	});
 };
 
+export type App = {
+	name: string;
+	commit: string;
+	releaseId: number;
+	releaseVersion: string;
+	appId?: string;
+	uuid?: string;
+	services: {
+		[id: number]: AppService;
+	};
+	volumes: any;
+	networks: any;
+};
+
+export type AppService = {
+	imageId: number;
+	serviceName: string;
+	image: string;
+	running: boolean;
+	environment: Dictionary<string>;
+	labels: Dictionary<string>;
+	contract?: string;
+};
+
+export type LocalState = {
+	name: string;
+	config: Dictionary<string>;
+	apps: Dictionary<Partial<App>>;
+};
+
+async function buildAppFromRelease(
+	device: AnyObject,
+	application: AnyObject,
+	release: AnyObject,
+	config: Dictionary<string>,
+): Promise<App> {
+	let composition: any = {};
```

Comment:
Please type this better than `any`, at the barest minimum it should be `AnyObject`

Reply:
resolved in other PR